src/feat/mel-computations.h

```diff
@@ -69,6 +71,13 @@ struct MelBanksOptions {
     opts->Register("vtln-high", &vtln_high,
                    "High inflection point in piecewise linear VTLN warping function"
                    " (if negative, offset from high-mel-freq");
+    opts->Register("modified", &modified,
+                   "Modified MFCCs, based on paper 'An alternative to MFCCs for ASR' "
```

Comment:
Please update this documentation to be accurate and fix typos.  (the stuff about 1nt and 2nd formant isn't accurate any more, I believe).